src/statsd_client.cpp

```diff
@@ -64,16 +64,8 @@ StatsdClient::StatsdClient(const std::string& host, const std::string& nodename,
         return;
     }
 
-    SOCKET hSocket = socket(AF_INET, SOCK_DGRAM, IPPROTO_UDP);
```

Comment:
nit: e2951fec6dc7e3de5a8a77276d0b97b4d4435130 should be a feat

Reply:
Resolved in latest push!